blocks/ads-block/features/ads/default.jsx

```diff
@@ -33,7 +33,6 @@ const StyledAdUnit = styled.div`
 `;
 
 const ArcAd = (props) => {
-  if (typeof window === 'undefined') return null;
```

Comment:
was this change deliberate?

Reply:
Needs to return from the server so we can place the element on the page to reserve the space

Reply:
wait, we need this

Reply:
yep just missed your comment. was thinking about this more heh

Reply:
Just for context, I originally added the window check to prevent the ads code from executing server-side and rendering a blank ad placeholder element. That being said, I was planning on refactoring this because I think it breaks the rules of hooks since it returns before the `useState` calls (but then I left the team and never got a chance to actually do that). 

I don't _think_ removing it should cause any issues since I don't think the `arcads` library should actually make ad requests when server-side. I'm not 100% sure on that part though. There might need to be a check to prevent `arcads` from being initialized & registering ad units when server-side if not preventing the `ads-block` from rendering server-side. Not completely sure without testing that though.

Reply:
ok cool, we'll move forward with your idea. @matthewroach can you remove the rule of hooks ignore in the first line

Reply:
no problem @JackHowa 🙂